### PR TITLE
add in extra badges to the badge export

### PR DESCRIPTION
**Pull request by binary1230**
simplify badge export now that #2143 is in

### CODE REVIEW

**Comment by binary1230** [comment]
this creates a bunch of extra badges in the staff badge export for staff badges that aren't assigned (usually part of groups)

File: uber/site_sections/summary.py [hunk 1]
```diff
@@ -157,13 +157,21 @@ def printed_badges_staff(self, out, session):
 
         # part 1, include only staff badges that have an assigned name
         uber.reports.PersonalizedBadgeReport().run(out, session,
-            sa.Attendee.badge_type == c.STAFF_BADGE,
-            sa.Attendee.badge_num != None,
+            Attendee.badge_type == c.STAFF_BADGE,
+            Attendee.badge_num != None,
             order_by='badge_num')
 
         # part 2, include a bunch of extra badges so we have some printed
```
Comment (kitsuta) [inline uber/site_sections/summary.py]: I don't think we need both this and part 3 - they both accomplish the same thing. At the very least, maybe bump this down to 5 or something, for the much rarer case of someone being totally forgotten (vs the less-rare case of a department head not getting all their staff badges in on time)

File: uber/site_sections/summary.py [hunk 2]
```diff
@@ -157,14 +157,21 @@ def printed_badges_staff(self, out, session):
...
         uber.reports.PersonalizedBadgeReport().run(out, session,
-            sa.Attendee.badge_type == c.STAFF_BADGE,
-            sa.Attendee.badge_num != None,
+            Attendee.badge_type == c.STAFF_BADGE,
+            Attendee.badge_num != None,
             order_by='badge_num')
 
-        # part 2, include a bunch of extra badges so we have some printed
+        # part 2, include unassigned group badges that are staff.  these don't have badge numbers
+        minimum_safety_amount = 5
```
Comment (kitsuta) [inline uber/site_sections/summary.py]: Sorry, not quite right here!

What we'd want is:
Always a minimum of 10 'extra' badges
The unassigned badges + 5

So what you'd wanna do here is take out this var, then after your extra_count down below, add:

`extra_count = min(10, extra_count + 5)`

And then you can take out the `max()` line in the final `badge_range` assignment

File: uber/site_sections/summary.py [hunk 3]
```diff
@@ -138,34 +138,39 @@ def dept_head_emails(self, out, session):
...
 
     @csv_file
     def printed_badges_minor(self, out, session):
-        uber.reports.PrintedBadgeReport(badge_type=c.CHILD_BADGE).run(out, session)
+        uber.reports.PrintedBadgeReport(badge_type=c.CHILD_BADGE, badge_type_name='Minor').run(out, session)
 
     @csv_file
     def printed_badges_staff(self, out, session):
 
         # part 1, include only staff badges that have an assigned name
```
Comment (kitsuta) [inline uber/site_sections/summary.py]: This comment is now wrong. :P NBD though

**Pull request merged**